src/ringmesh/geogram_extension/geogram_mesh.cpp

```diff
@@ -40,6 +40,7 @@ namespace RINGMesh
 {
     void register_geogram_mesh()
     {
+        std::cout << "je passe au debut " << std::endl;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of "::std::cout" by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS106)